gateway/modules/eventing/handle_staged.go

```diff
@@ -132,7 +132,9 @@ func (m *Module) processStagedEvent(eventDoc *model.EventDocument) {
 		// Reaching here means the event was successfully processed. Let's simply return
 		return
 	}
-
+	if err := m.generateDLQEventRequest(eventDoc); err != nil {
+		logrus.Errorf(err.Error())
```

Comment:
Use utils.LogErrror() with message "Couldn't create DLQ event for event id %v" eventdoc.id